Sources/Sentry/Transactions/SentrySpanContext.h

```diff
@@ -0,0 +1,37 @@
+#import "SentryDefines.h"
+#import "SentrySerializable.h"
+#import <Sentry/SentrySpanId.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+@class SentryId;
+
+NS_SWIFT_NAME(SpanContext)
+@interface SentrySpanContext : NSObject
+
+@property (nonatomic, strong) SentryId *traceId;
+
+@property (nonatomic, strong) SentrySpanId *spanId;
+
+@property (nonatomic, strong) SentrySpanId *_Nullable parentSpanId;
+
+@property (nonatomic) BOOL sampled;
+
+@property (nonatomic, copy) NSString *_Nullable operation;
+
+@property (nonatomic, copy) NSString *_Nullable spanDescription;
```

Comment:
why not only `description`

Reply:
description is a Objc NSObject method already. Like toString from java/.NET. I guess we could override it and create the setDescription, I dont know if  I like this option.

Reply:
oh I see its a reserved word, well that's ok then I'd say, to keep as it is